google_analytics/tests/test_ga.py

```diff
@@ -150,16 +150,45 @@ def test_build_ga_params_for_title_encoding(self):
     def test_build_ga_params_for_user_id(self):
         request = self.make_fake_request('/somewhere/')
 
-        ga_dict_without_uid = build_ga_params(
+        ga_dict_without_dr = build_ga_params(
```

Comment:
Why is it necessary to rename this variable? Is the old name no longer accurate?